### ISSUE TITLE
Feature Request: Let us set adb.exe to use with an environment variable or a setting.

### ISSUE THREAD

**Issue by esteimle** [koush/vysor.io#896]
I frequently run into the vysor wants to use adb version 41 but 40 is running issue. I know it is a version mismatch that causes it but with scrcpy I can solve this by making scrcpy use adb version 40 with an environment variable.  Could we have something similar for vysor?

Thank you

**Comment by koush** [comment]
Now available via a setting:

<img width="254" alt="image" src="https://user-images.githubusercontent.com/73924/128751946-014dd97d-6175-4789-b17c-2c9f1342a1db.png">


You can specific the path (minus the executable name), and optionally arguments (so you can connect to remote adb daemons). So, to use my android sdk version setup would look like:

```json
      "adb": {
        "path": "/Volumes/Dev/android-sdk/platform-tools",
        "arguments": [],
      },
```

**Comment by esteimle** [comment]
I can't get this to work in windows. I opened the settings and found the android section then I tried to add it as you said. At first I got lots of Json errors until finally I got it to take this:

    "adb": {
      "path": "C:\\Users\\my_username\\AppData\\Local\\Android\\sdk\\platform-tools\\"
    },

I killed all vysor tasks and restarted but still it is using your adb version

No matter how I tried to add "arguments": [], I got json errors from the settings window.
